ocaml/xapi/xapi_mgmt_iface.mli

```diff
@@ -41,5 +41,5 @@ val rebind : __context:Context.t -> unit
 (** Start a server thread on the given HIMN address if the server is not yet running *)
 val enable_himn : __context:Context.t -> addr:string -> unit
 
-(** Restart stunnel to make it pick up a change to host.ssl_legacy *)
+(** Restart stunnel *)
```

Comment:
Is this function obsolete now? If it has no other callers we can delete it too

Reply:
This function may be useful to reload certificates, don't delete it just yet.

Reply:
yes